packages/fluent-ui/src/FieldTemplate/FieldTemplate.tsx

```diff
@@ -26,7 +26,7 @@ export default function FieldTemplate<
   return (
     <WrapIfAdditionalTemplate {...props}>
       {children}
-      {displayLabel && rawDescription && <Text>{rawDescription}</Text>}
+      {displayLabel && description && <Text>{description}</Text>}
```

Comment:
`rawDescription` should continue to be used as the condition since it will be null when there is no description provided. This should cause most of the need to update the snapshots to go away when the description is an empty React element.
```suggestion
      {displayLabel && rawDescription && <Text>{description}</Text>}
```